cypress/integration/common/search.js

```diff
@@ -85,6 +85,26 @@ Then(
   }
 );
 
+Then("I should see the search results page", () => {
```

Comment:
> [<img alt="Tirokk" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/Tirokk) **Authored by [Tirokk](https://github.com/Tirokk)**
_<time datetime="2020-06-22T13:17:30Z" title="Monday, June 22nd 2020, 3:17:30 pm +02:00">Jun 22, 2020</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -85,6 +85,26 @@ Then(
   }
 );
 
+Then("I should see the search results page", () => {
```


Nice to have e2e tests! 😍